tests/tests.js

```diff
@@ -59,6 +59,21 @@ describe("htmlbars-inline-precompile", function() {
     }, /placeholders inside a tagged template string are not supported/);
   });
 
+  it("moduleName is supplies to the precompiler", function() {
+    var fullpath = path.join(__dirname, 'fixtures', 'hello-world.js');
+
+    var precompile = function(template, options) {
+      assert.equal(options.moduleName, fullpath);
```

Comment:
> Ember uses it internally for hinting at deprecation warnings in a template. I use it for building scoped CSS modules.

Thanks for the heads up!

Ok, waiting seems like a good idea. Thanks for looking into this!